container-storage-setup.spec

```diff
@@ -1,6 +1,6 @@
 %global project container-storage-setup
 %global git0 https://github.com/projectatomic/%{repo}
-%define csslibdir %{_prefix}/lib/container-storage-setup
+%global csslibdir %{_prefix}/share/container-storage-setup
```

Comment:
I think we can continue keeping this at %{_prefix}/lib. Changing %define to %global was what would make one of the complains go away.